cli/printer/text_test.go

```diff
@@ -9,15 +9,29 @@ import (
 func ExampleTextPrintDeploys() {
 	printer := &TextPrinter{}
 	deploys := []*models.Deploy{
-		{DeployID: "id1", DeployName: "name1", Version: "1"},
-		{DeployID: "id2", DeployName: "name2", Version: "2"},
+		{
+			Compatibilities: []string{models.DeployCompatibilityStateless},
+			DeployID:        "id1",
```

Comment:
[nit] 
```
dpl_id
dpl_name
```

Reply:
I'm tempted to leave these as-is, instead of changing the entity IDs for every single test case in this file. It seems like a lot of busywork for barely any benefit.